templates/blocks/categories/categories_dropdown_vertical.tpl

```diff
@@ -3,87 +3,135 @@
 {assign var="name" value="category"}
 {assign var="childs" value="subcategories"}
 
-<nav class="navbar navbar-inverse has-submenu no-margin-nav" role="navigation">
-    <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar_{$block.block_id}" aria-expanded="false">
-
-            <span class="sr-only">Toggle navigation</span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-            <span class="icon-bar"></span>
-        </button>
-        <a class="navbar-brand visible-xs-inline-block" href="#" data-toggle="collapse" data-target="#navbar_{$block.block_id}">{__("category")}</a>
-    </div>
-
-    <div class="collapse navbar-collapse" id="navbar_{$block.block_id}">
-        <ul class="nav nav-pills nav-stacked">
-            {foreach from=$items item="item1" name="item1"}
-                {assign var="item1_url" value=$item1|fn_form_dropdown_object_link:$block.type}
-                {assign var="unique_elm_id" value=$item1_url|md5}
-                {assign var="unique_elm_id" value="topmenu_`$block.block_id`_`$unique_elm_id`"}
-                {if $subitems_count}
-                {/if}
-                <li class="dropdown {if $item1.active || $item1|fn_check_is_active_menu_item:$block.type} active{/if}{if $item1.class} {$item1.class}{/if}">
-                    <a {if $item1_url} href="{$item1_url}"{/if} class="dropdown-toggle">{$item1.$name}{if $item1.$childs} <span class="caret"></span>{/if}</a>
+<style>
+    .vertical-improve .nav > li > .nav {
+        padding: 10px 10px;
+        border: 1px solid #eee;
+        margin-left: -1px;
+        margin-right: -1px;
+        margin-bottom: 5px;
+    }
 
+    .vertical-improve a {
+        cursor: pointer;
+    }
+
+    .vertical-improve .active > a {
+        background-color: #eee;
+    }
+
+    .vertical-improve a[aria-expanded=true] {
+        background-color: #eee;
+        margin: 0 -1px;
+    }
+
+    .vertical-improve a[aria-expanded=true] .caret {
+        transform: rotate(180deg)
+    }
+
+    .vertical-improve {
+        border: 1px solid #ddd;
+        border-radius: 5px;
+    }
+
+    .vertical-improve-heading {
+        background-color: #ddd;
+
+        position: relative;
+        display: block;
+        padding: 10px 15px;
+        margin: 0;
+
+        border-bottom: 1px solid #ddd;
+
+        font-weight: 700;
+    }
+</style>
+
+{$blk_name = $block.name}
+
+<nav class="vertical-improve">
+    <p class="vertical-improve-heading">{$blk_name}</p>
+    <ul class="nav">
+        {foreach from=$items item="item1" name="item1"}
```

Comment:
http://docs.cs-cart.com/4.6.x/developer_guide/core/coding_standards/php.html#names-of-variables-array-keys-and-class-properties
```tpl
{foreach $items as $item_level_1}
```